src/GraphQL/Execution/ExecutionResult.php

```diff
@@ -5,7 +5,8 @@
 use Drupal\Core\Cache\CacheableDependencyInterface;
 use Drupal\Core\Cache\RefinableCacheableDependencyTrait;
 
-class ExecutionResult extends \GraphQL\Executor\ExecutionResult implements CacheableDependencyInterface {
+//@codingStandardsIgnoreStart
```

Comment:
why do we need to ignore here? Please add a code comment.

Reply:
ah, so we have 2 classes with the same name here. You need to declare
```php
use GraphQL\Executor\ExecutionResult as LibraryExecutionResult;
```

and then
```php
class ExecutionResult extends LibraryExecutionResult implements CacheableDependencyInterface {
```